src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -300,13 +288,13 @@ const styles = StyleSheet.create({
     position: 'absolute',
     opacity: opacityHeaderAnimation
   }),
-  image: {
+  image: (followType) => ({
     width: normalize(48),
     height: normalize(48),
     borderRadius: normalize(24),
-    backgroundColor: 'lightgrey',
+    backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
     marginRight: 8
-  },
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style for the image now depends on the `followType`. This change seems fine as long as the `followType` prop is always provided and has a valid value.
```diff
- image: {
+ image: (followType) => ({
    width: normalize(48),
    height: normalize(48),
    borderRadius: normalize(24),
-   backgroundColor: 'lightgrey',
+   backgroundColor: followType === 'signed' ? 'lightgrey' : COLORS.anon_secondary,
    marginRight: 8
- },
+ }),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->